docs/data/tree-view/rich-tree-view/ordering/ordering.md

```diff
@@ -0,0 +1,68 @@
+---
+productId: x-tree-view
+title: Rich Tree View - Ordering
+components: RichTreeView, TreeItem2
+packageName: '@mui/x-tree-view'
+githubLabel: 'component: tree view'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/
+---
+
+# Rich Tree View - Ordering [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+<p class="description">Drag and drop your items to reorder them.</p>
+
+:::success
+To be able to reorder items, you first have to enable the `indentationAtItemLevel` experimental feature.
+When this flag is enabled, the indentation of nested items is applied by the item itself instead of its ancestors.
+This allows correctly placing the drag & drop overlay.
+
+You can enable it as follows:
+
+```tsx
+<RichTreeViewPro
+  items={ITEMS}
+  experimentalFeatures={{ indentationAtItemLevel: true }}
+/>
+```
+
+If you are building your custom Tree Item, make sure that you do not apply any custom `padding-left` on your Tree Item Content component.
+If you want to change the indentation value, you can use the `itemChildrenIndentation` prop on the Tree View component.
+:::
+
+## Enable drag & drop re-ordering
+
+You can enable the drag & drop re-ordering of items by setting the `itemsReordering` prop to `true`:
+
+{{"demo": "DragAndDrop.js"}}
+
+## Limit the re-ordering
+
+By default, all the items are reorderable.
+You can prevent the re-ordering of some items using the `isItemReorderable` prop.
+The following example shows how to only allow re-ordering of the leaves using the [`getItemOrderedChildrenIds`](/x/react-tree-view/rich-tree-view/items/#get-an-items-children-by-id) API method.
+
+{{"demo": "OnlyReorderLeaves.js"}}
```

Comment:
The example of reordering leaves is not great because it is here to showcase `isItemReorderable` so it limits which items you can re-order, not where you can re-order them :/
Which means right now you can even drop `@mui/x-data-grid` inside `@mui/x-date-pickers`...


Is the little red cursor in the video something that is really visible in your browser?
Because since you are able to drop in any item, I'm surprised this is visible...

Reply:
> Is the little red cursor in the video something that is really visible in your browser?

Yes, in both Chrome and Firefox, that's why I said that it's confusing.